yal/mainmenu.py

```diff
@@ -217,6 +218,18 @@ async def next_(choice: Choice):
         )
 
     async def state_contentrepo_page(self):
+        msisdn = utils.normalise_phonenumber(self.inbound.from_addr)
+        whatsapp_id = msisdn.lstrip("+")
+        timestamp = datetime.now() + timedelta(minutes=10)
```

Comment:
There is a `get_current_datetime` method in yal.utils
maybe we should use that?